### PR TITLE
feat: Image add PreviewGroup component to support multiple images preview

### PR DESCRIPTION
**Pull request by shaodahong**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |Image add PreviewGroup component to support multiple images preview|
| 🇨🇳 Chinese | Image 增加 PreviewGroup 组件支持多张图片预览 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/image/demo/previewGroup.md](https://github.com/shaodahong/ant-design/blob/image-preview-group/components/image/demo/previewGroup.md)

### CODE REVIEW

File: components/image/index.tsx [hunk 1]
```diff
@@ -11,4 +16,6 @@ const Image: React.FC<ImageProps> = ({ prefixCls: customizePrefixCls, ...otherPr
 
 export { ImageProps };
 
+Image.PreviewGroup = PreviewGroup;
```
Comment (afc163) [inline components/image/index.tsx:19]: Image.PreviewGroup 还是 Image.Group 好一点？

Comment (shaodahong) [reply components/image/index.tsx:19]: 一开始我提的想法是 `Image.Group`

后来改掉的

https://github.com/react-component/image/pull/22#issuecomment-681656967

**Comment by shaodahong** [comment]
顶部的操作栏在刚打开出现动画时有点别扭，应该在 `transitionEnd` 后显示，dialog 感觉要加个类似的 api 支持下

File: components/image/index.tsx (hunk 1 above)
Comment (zombieJ) [reply components/image/index.tsx:19]: 我比较倾向于先保留 Group，用 PreviewGroup。

**Comment by zombieJ** [comment]
> 顶部的操作栏在刚打开出现动画时有点别扭，应该在 `transitionEnd` 后显示，dialog 感觉要加个类似的 api 支持下

这个之后再调就行了。这个 PR 可以先合

**Review by zombieJ — APPROVED** [review]

**Pull request merged**